set-based-validation/src/main/resources/META-INF/services/org.axonframework.messaging.annotation.ParameterResolverFactory

```diff
@@ -0,0 +1 @@
+io.axoniq.dev.samples.resolver.EmailAlreadyExistsResolverFactory
```

Comment:
As the `EmailAlreadyExistsResolverFactory` has been annotated with `@Component`, I would assume this service loader file is not necessary anymore.